data/json/items/magazine/545x39.json

```diff
@@ -3,7 +3,7 @@
     "id": "ak74mag",
     "looks_like": "stanag30",
     "type": "MAGAZINE",
-    "name": { "str": "AK-74M magazine" },
+    "name": { "str": "AK-74M 30-round magazine" },
     "description": "A standard magazine made for the AK-74M rifle, made of steel-reinforced polymer.  Holds 30 rounds.",
```

Comment:
```suggestion
    "description": "A standard magazine made for the AK-74M rifle, made of steel-reinforced polymer. Holds 30 rounds.",
```

Reply:
The project standard is two spaces after periods since we use fixed-width fonts.